src/OpenAL/OpenToolkit.OpenAL/ALC/ALContextHandle.cs

```diff
@@ -0,0 +1,52 @@
+﻿// ALContextHandle.cs
+//
+// Copyright (C) 2020 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    public struct ALContextHandle : IEquatable<ALContextHandle>
+    {
+        public static readonly ALContextHandle Zero = new ALContextHandle(IntPtr.Zero);
+
+        public IntPtr Handle;
```

Comment:
Instead of exposing Handle, this struct could use implicit type conversion, and ALContextHandle is what it is. Just a thought.

Reply:
There is an implicit cast now